<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>네이버 자동완성 키워드 추출기</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>서울남산 장체B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>검색어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyword_line"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>상태 메시지</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="result_view"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="extract_btn">
         <property name="text">
          <string>추출</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>저장</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_btn">
         <property name="text">
          <string>리셋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn">
         <property name="text">
          <string>종료</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="subject_label">
     <property name="font">
      <font>
       <family>서울남산 장체B</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55aa00;&quot;&gt;네이버 자동완성&lt;/span&gt; 키워드 추출기&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
